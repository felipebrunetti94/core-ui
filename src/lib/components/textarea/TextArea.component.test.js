import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/textarea/__snapshots__"
});
