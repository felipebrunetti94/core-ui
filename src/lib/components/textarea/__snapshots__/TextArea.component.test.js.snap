// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots TextArea Default 1`] = `
<div
  className="sc-bwzfXH hilSGa"
>
  <h3
    className="sc-htpNat bnCVRm"
  >
    Text Area without label
  </h3>
  <div
    className="sc-textarea sc-bdVaJa chipxc"
  >
    <textarea
      cols={200}
      id="text"
      rows={10}
    />
  </div>
  <h3
    className="sc-htpNat bnCVRm"
  >
    Text Area with label
  </h3>
  <div
    className="sc-textarea sc-bdVaJa chipxc"
  >
    <textarea
      cols={50}
      id="text"
      placeholder="Text area input"
      rows={10}
    />
  </div>
  <h3
    className="sc-htpNat bnCVRm"
  >
    Text Area with error
  </h3>
  <div
    className="sc-textarea sc-bdVaJa chipxc"
  >
    <textarea
      cols={50}
      error="error"
      id="text"
      placeholder="Text area input"
      rows={10}
    />
  </div>
  <h3
    className="sc-htpNat bnCVRm"
  >
    Text Area disabled
  </h3>
  <div
    className="sc-textarea sc-bdVaJa chipxc"
  >
    <textarea
      cols={50}
      disabled={true}
      id="text"
      placeholder="Text area input"
      rows={10}
    />
  </div>
</div>
`;
